csuntechlab/iSTART
# STR-278 -- Update Admin Styling

# Description
Finished up the rest of the admin page styling & UX fixes.
  
# Testing
1. Visit the `admin` page.
2. Check to ensure **import** and **submission** functionality still works.
3. Ensure the new design doesn't break the page.
